eks/verify_kubera_on_eks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+ 
+set +e
+
+## Install helm
+curl -fsSL -o get_helm.sh \
+https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+
+chmod 700 get_helm.sh
+./get_helm.sh
+helm version
+
+## Creates secret this secret will help us to pull images from registry
+kubectl create secret docker-registry directoronprem-registry-secret \
+--docker-server=registry.mayadata.io \
+--docker-username="$DOCKER_USER" --docker-password="$DOCKER_PASS"
+kubectl get secret
+
+## To access kubera installation approved solution is to use an external Load Balancer 
+## with DNS to point to the hostname you are using for server.url
+IP=$(sudo kubectl get nodes -o wide --no-headers | awk '{print $6}' | head -n 1)
+URL=http://$IP
+helm repo add kubera https://charts.mayadata.io/
```

Comment:
In order to user particular branches of this private repo, we might need to provide authentication.
e.g. https://github.com/mayadata-io/kubera-charts/tree/2.0
https://github.com/mayadata-io/kubera-charts/tree/master
Above will need us to provide auth

Reply:
Please check if we can avoid git clone of chart repo & just use helm to get the stuff done